### DISCUSSION TITLE
Taskbar clock customization ?

### DISCUSSION THREAD

**Discussion by sypqys** [ramensoftware/windhawk-mods#431] [Q&A] [answered]
Hi !

I want 24 h
And more bigger the police... And "bold" like this 👍
"**dimanche 17 décembre 2023 .....**" 
![pAMvUIOIzq](https://github.com/ramensoftware/windhawk-mods/assets/27967268/8afffca5-cbd7-40b1-8ab0-2cc48d7b359e)

**Comment by m417z** [comment, marked as answer]
> I want 24 h

Use `HH` instead of `hh` in "Time format". See:
https://learn.microsoft.com/fr-fr/windows/win32/api/datetimeapi/nf-datetimeapi-gettimeformatex#remarks

> And more bigger the police...

Bigger text? See "Font size" in settings.

> And "bold" like this 👍

See "Font weight" in settings.

**Reply by sypqys** [reply]
Thank you very much again !

have a nice day !
